main-app/app/scripts/checkingNumbers/numberService.js

```diff
@@ -0,0 +1,54 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.NumberService')
+        .service('numberService',['bingoModel', function (bingoModel) {
+            var me = this;
+            me.linePrize = true;
+
+            me.findNumberInTicket = function (calledNumber) {
```

Comment:
If you arrange the objects correctly - you only need to check one index on each line (effectively the 10s column. 20s column etc...)